packages/deployer/utils/deployments.js

```diff
@@ -108,6 +109,21 @@ function getDeploymentFolder(info) {
   return path.resolve(folder, network, instance);
 }
 
+/**
+ * Deploys a system.
+ * Designed for programmatic use in tests.
+ * @param {DeploymentInfo} info See DeploymentInfo above
+ * @returns {string} The path of the folder
+ */
+async function deploySystem(deploymentInfo, customOptions = {}) {
+  await hre.run(TASK_DEPLOY, {
+    ...deploymentInfo,
+    noConfirm: true,
+    quiet: true,
+    ...customOptions,
+  });
+}
+
```

Comment:
IMHO, we could just call `hre.run(TASK_DEPLOY, { ... })` instead of adding this helper here. Also, it adds the usage of the global `hre` to this util.